### ISSUE TITLE
Unable to create the 'tract_profile_plots' output folder

### ISSUE THREAD

**Issue by tpatpa** [yeatmanlab/pyAFQ#1034]
I have tried with both version 1.1 and dev version 1.2.dev4564630937 and I am unable to create the 'tract_profile_plots' folder.
Is this not part of the output anymore?

I ran:
```
myafq.export_all(afqbrowser=False,xforms=False)
```
which returns:
```
WARNING:AFQ:To make plots in pyAFQ, you will need to install the relevant plotting software packages. You can do that by installing pyAFQ with `pip install AFQ[plot]`
```

I have previously used:
```
myafq.tract_profile_plots[subject]
```
Which now returns:
```
{
	"name": "AttributeError",
	"message": "'GroupAFQ' object has no attribute 'tract_profile_plots'",
	"stack": "---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[11], line 1
----> 1 myafq.tract_profile_plots[subject]

AttributeError: 'GroupAFQ' object has no attribute 'tract_profile_plots'"
}
```
Also tried:
```
fig_files = myafq.export("tract_profile_plots")[subject]
```
Which returns:
```
{
	"name": "ImportError",
	"message": "To make plots in pyAFQ, you will need to install the relevant plotting software packages.You can do that by installing pyAFQ with `pip install AFQ[plot]`",
	"stack": "---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
File ~/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py:17
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=15'>16</a> try:
---> <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=16'>17</a>     from pingouin import intraclass_corr, corr
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=17'>18</a>     import seaborn as sns

ModuleNotFoundError: No module named 'pingouin'

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
Cell In[15], line 1
----> 1 fig_files = myafq.export(\"tract_profile_plots\")[subject]

File ~/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/api/group.py:434, in GroupAFQ.export(self, attr_name, collapse)
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/api/group.py?line=431'>432</a>         to_calc_list.append((subject, session))
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/api/group.py?line=432'>433</a>     else:
--> <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/api/group.py?line=433'>434</a>         results[subject][session] = wf_dict[attr_name]
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/api/group.py?line=435'>436</a> # if some need to be calculated, do those in parallel
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/api/group.py?line=436'>437</a> if to_calc_list:

File ~/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py:470, in IMap.__getitem__(self, k)
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=467'>468</a> effval = self.efferents.get(k, self)
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=468'>469</a> if effval is self:
--> <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=469'>470</a>     self._run_node(self.plan.efferents[k])
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=470'>471</a>     return self.efferents[k]
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=471'>472</a> else:

File ~/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py:534, in IMap._run_node(self, node)
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=530'>531</a>     else: cpath = None
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=531'>532</a>     #except Exception: #(h, found) = (None, False)
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=532'>533</a> # ensure we have a result
--> <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=533'>534</a> if not found: res = node(self)
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=534'>535</a> # process the result:
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=535'>536</a> effs = reduce(lambda m,v: m.set(v[0],v[1]), six.iteritems(res), self.efferents)

File ~/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py:91, in Calc.__call__(self, *args, **kwargs)
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=88'>89</a>         raise ValueError('required parameter %s not given to %s' % (name, self.name))
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=89'>90</a>     args.append(opts[name])
---> <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=90'>91</a> result = self.function(*args)
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=91'>92</a> if is_map(result):
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/pimms/calculation.py?line=92'>93</a>     if len(result) != len(self.efferents) or not all(e in result for e in self.efferents):

File ~/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py:366, in plot_tract_profiles(base_fname, scalars, tracking_params, segmentation_params, segmentation_imap)
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=358'>359</a> @pimms.calc(\"tract_profile_plots\")
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=359'>360</a> def plot_tract_profiles(base_fname, scalars, tracking_params,
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=360'>361</a>                         segmentation_params, segmentation_imap):
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=361'>362</a>     \"\"\"
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=362'>363</a>     list of full paths to png files,
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=363'>364</a>     where files contain plots of the tract profiles
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=364'>365</a>     \"\"\"
--> <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=365'>366</a>     from AFQ.viz.plot import visualize_tract_profiles
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=366'>367</a>     start_time = time()
    <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/tasks/viz.py?line=367'>368</a>     fnames = []

File ~/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py:24
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=21'>22</a>     import IPython.display as display
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=22'>23</a> except (ImportError, ModuleNotFoundError):
---> <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=23'>24</a>     raise ImportError(vut.viz_import_msg_error(\"plot\"))
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=25'>26</a> __all__ = [\"visualize_tract_profiles\", \"visualize_gif_inline\"]
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=28'>29</a> def visualize_tract_profiles(tract_profiles, scalar=\"dti_fa\", ylim=None,
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=29'>30</a>                              n_boot=1000,
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=30'>31</a>                              file_name=None,
     <a href='file:///Users/m161121/opt/anaconda3/envs/DWI_newPython/lib/python3.8/site-packages/AFQ/viz/plot.py?line=31'>32</a>                              positions=vut.POSITIONS):

ImportError: To make plots in pyAFQ, you will need to install the relevant plotting software packages.You can do that by installing pyAFQ with `pip install AFQ[plot]`"
}
```

This is on a fresh conda environment with Python 3.8.18.

**Comment by tpatpa** [comment]
This is a local issue with  `pip install AFQ[plot]`, the issue can be closed.
